src/pages/guides/getting_started/changelog.md

```diff
@@ -22,6 +22,26 @@ contributors:
 
 # Changelog
 
+## 2025-11-24
+
+### Code Playground Updates
+
+The Adobe Express Code Playground has received a major update with the release of a new set of features and improvements, which this site now reflects.
+
+- The Code Playground now supports [Cloud-based session persistence](./code-playground-workflow.md#session-management) (instead of the previous local storage) allowing you to save your work and resume where you left off everywhere. The new **More** menu contains items to manage your sessions (rename or delete), or star new ones.
+- Relevant Code Blocks (e.g. in the [How-to guides](../learn/how_to/use_text.md)) now feature a **Try in playground** button that opens a new Playground Script session with the code pre-filled for easier testing.
+- The experience can now be launched directly from the documentation via a new **Code Playground button in the top navigation bar** (present in the How-to guides, Hello World!, Code Playground pages, and in any other sections where it makes sense), or by clicking directly on [this link](https://www.adobe.com/go/addon-playground). You can bookmark links that point to either the [Script](https://www.adobe.com/go/addon-playground?executionMode=script) or [Add-on](https://www.adobe.com/go/addon-playground) Mode.
```

Comment:
Lets make this as first point.

Reply:
'Open Code Playground with a single click (https://www.adobe.com/go/addon-playground) or find the **Code Playground** button in the top navigation bar. You can also bookmark links that point to either the [Script](https://www.adobe.com/go/addon-playground?executionMode=script) or [Add-on](https://www.adobe.com/go/addon-playground) Mode.'